Makefile

```diff
@@ -289,9 +289,20 @@ validate-python-sdk:
 validate-client-java:
 	git diff --quiet -- clients/java || (echo "Modification verification failed! java client"; false)
 
+validate-python-wrapper:
+	sphinx-apidoc -o clients/python-wrapper/docs clients/python-wrapper/lakefs sphinx-apidoc --full -A 'Treeverse' -eq && git diff --quiet -- clients/python-wrapper || (echo 'Modification verification failed! python wrapper client'; false)
```

Comment:
Can you split into two lines - one for the `sphinx-apidoc` and the second for the git validation?
Make will break in case the apidoc generation fails and the error will clear.